src/components/DraggableList/types.ts

```diff
@@ -0,0 +1,21 @@
+import type {DraggableChildrenFn} from 'react-beautiful-dnd';
+import type {RenderItemParams as OriginalRenderItemParams} from 'react-native-draggable-flatlist';
+
+type DraggableListData<T> = {
+    data: T[];
+};
+
+type DraggableListProps<T> = {
+    keyExtractor: (item: T, index: number) => string;
+    renderItem: (params: RenderItemParams<T>) => React.ReactNode;
+    onDragEnd?: (params: DraggableListData<T>) => void;
+    onDragBegin?: () => void;
+    onPlaceholderIndexChange?: (placeholderIndex: number) => void;
+    renderClone?: DraggableChildrenFn;
+    shouldUsePortal?: boolean;
+    ListFooterComponent?: React.ReactElement;
+} & DraggableListData<T>;
+
+type RenderItemParams<T> = OriginalRenderItemParams<T>;
```

Comment:
why the need for renaming?

Reply:
I solely used `RenderItemParams` but it didn't give an error

<img width="400" alt="Screenshot 2023-10-03 at 11 25 09 AM" src="https://github.com/Expensify/App/assets/98560306/1872d5c2-ddc4-431f-955f-3387d746951c">